tuning/config/peft_config.py

```diff
@@ -1,21 +1,18 @@
 from dataclasses import dataclass, field
 from typing import List
 
-
 @dataclass
-class lora_config:
+class LoraConfig:
     r: int = 8
     lora_alpha: int = 32
     target_modules: List[str] = field(default_factory=lambda: ["q_proj", "v_proj"])
     bias = "none"
-    task_type: str = "CAUSAL_LM"
```

Comment:
since the repo only supports CausalLMs we need not expose task_type to user as an argument.